config/nightwatch.js

```diff
@@ -19,6 +19,9 @@ module.exports = {
   test_workers: false,
   test_settings: {
     'default': {
+      globals: {
+        buildtype: process.env.BUILDTYPE
```

Comment:
OK, I removed the `nightwatch.js` edits and am checking the buildtype outside of the test creation, so the tests themselves don't need to know the buildtype.

I do like the idea of tracking individual unlaunched features, because then we can grep for that key when launching the feature and make sure there are no lingering disabled tests. I'll move this conversation into a separate issue so it doesn't get lost when this PR is merged.